backend/battling/views.py

```diff
@@ -13,3 +13,72 @@ def home(request):
     gamer = Battle.objects.all()
     return render(request, 'battling/home.html', { 'gamer' : gamer})
 
+def get_pokemon_from_api(poke_name):
+    url = urljoin(settings.POKE_API_URL, poke_name)
```

Comment:
I think you should convert `poke_name` into a lower-cased string since PokéApi requires that.

i.e.: If I type `Flareon` I get an error while if I type `flareon` it completes the request.

Reply:
Ok, nice! Will do :D The other two errors you pointed out below, they are fixed already, I just didn't commit them yet, because I needed to get a PR for this one first. Thanks, Renato!!!